cirq-core/cirq/contrib/qasm_import/_parser.py

```diff
@@ -499,6 +501,15 @@ def p_measurement(self, p):
             ops.MeasurementGate(num_qubits=1, key=creg[i]).on(qreg[i]) for i in range(len(qreg))
         ]
 
+    # reset operations
+    # reset : RESET qarg
+
+    def p_reset(self, p):
+        """reset : RESET qreg ';'"""
+        qreg = p[2]
+
+        p[0] = [ops.ResetChannel().on(qreg[i]) for i in range(len(qreg))]
```

Comment:
Should there be some sort of check here to ensure the reset is applied on a qubit that is actually defined in the quantum register? E.g. should some error be raised when trying to parse the following QASM program?
```qasm
OPENQASM 2.0;
include "qelib1.inc";
qreg q[1];
reset q[2];
```

Reply:
The example program already raises a QasmException at
https://github.com/quantumlib/Cirq/blob/e1e1730e8ab41cd6bcfc655b1a78993442797d7b/cirq-core/cirq/contrib/qasm_import/_parser.py#L459-L464